internal/venice-common/src/main/java/com/linkedin/venice/kafka/protocol/enums/MessageType.java

```diff
@@ -21,7 +21,8 @@
  */
 public enum MessageType implements VeniceEnumValue {
   PUT(0, Constants.PUT_KEY_HEADER_BYTE), DELETE(1, Constants.PUT_KEY_HEADER_BYTE),
-  CONTROL_MESSAGE(2, Constants.CONTROL_MESSAGE_KEY_HEADER_BYTE), UPDATE(3, Constants.UPDATE_KEY_HEADER_BYTE);
+  CONTROL_MESSAGE(2, Constants.CONTROL_MESSAGE_KEY_HEADER_BYTE), UPDATE(3, Constants.UPDATE_KEY_HEADER_BYTE),
+  GLOBAL_RT_DIV(4, Constants.GLOBAL_RT_DIV_KEY_HEADER_BYTE);
```

Comment:
I wonder if it is ok to not add it into KME.  I am thinking for example, Da Vinci and CDC, if the code is not upgraded, how can it be handling the global rt div? Or should we completely mute them from processing RT div message?
Will this controlled by store config or just server config?

Reply:
It's controlled by [server config](https://github.com/linkedin/venice/pull/1179/files#diff-f1babd8c1a4cf090b308fdba3add6bf82d484d969b126a35b1b406976056ed62R511).

Reply:
I mean, with this code change, we will eventually enable the feature to produce RT DIV snapshot to VT topic. This will be processed by follower, which also applicable to DVC. I know DVC won't be leader, but will the existing code handle unknown new msg type gracefully?

Reply:
You're right. The reader of the msg type (to be able to ignore it) has to be deployed before enabling this feature. i don't know if there is other way to walk around it, feel free to suggest alternatives.

Reply:
I didn't look very deep, but I think that the existing code will throw an exception when encountering an unknown message type.

Handling an unknown message type will be implemented in a future PR. Since we control the feature flag, we can choose not to enable the feature flag until all server instances support the new message type.

Reply:
We need to deploy the changes to all VT consumers before enabling this feature, which includes:
1. Server.
2. DaVinci.
3. ETL.
4. CDC.

Reply:
And I think it is better to treat the new KME type as a `Control Message`, and I would like to learn why if we can't do that.